go/vt/vtgate/executor_ddl_test.go

```diff
@@ -57,6 +55,8 @@ func TestDDLFlags(t *testing.T) {
 	}
 	for _, testcase := range testcases {
 		t.Run(fmt.Sprintf("%s-%v-%v", testcase.sql, testcase.enableDirectDDL, testcase.enableOnlineDDL), func(t *testing.T) {
+			executor, _, _, _, ctx := createExecutorEnv(t)
+			session := NewSafeSession(&vtgatepb.Session{TargetString: KsTestUnsharded})
 			enableDirectDDL = testcase.enableDirectDDL
 			enableOnlineDDL = testcase.enableOnlineDDL
```

Comment:
this change is not required anymore, correct?

Reply:
not really, but I still like it. cleaner to not re-use between tests

Reply:
`executor` is expected to be reused. This also helped in realising the caching issue with other statements.